src/test/regress/expected/upgrade_distributed_function_before.out

```diff
@@ -27,7 +27,7 @@ SELECT create_distributed_function('count_values(int)', '$1', colocate_with:='t1
 (1 row)
 
 -- make sure that the metadata synced before running the queries
-SELECT wait_until_metadata_sync();
+SELECT wait_until_metadata_sync(5000);
```

Comment:
I don't know whether we run these tests in valgrind. If so, then maybe make it 30 seconds, because things are always slower than expected. If not, this is fine.

Reply:
this specific test is used in pg upgrade job, so I dont think we use it in valgrind test, however for the other ones we can make it 30 seconds.